code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -408,7 +416,7 @@
 						blood_type = initial(blood_type)
 						dna_hash = initial(dna_hash)
 						fingerprint_hash = initial(fingerprint_hash)
-						access = initial(access)
+						access = initial_access // Initial() doesn't work

```

Comment:
Here and in New(), you probably want to set access to a copy of initial_access (`initial_access.Copy()`), rather than referencing it directly. With a direct reference like this, updating the access will also update initial_access.